components/tools/OmeroPy/src/omero/plugins/obj.py

```diff
@@ -297,38 +297,58 @@ def _configure(self, parser):
 
         self.exc = ExceptionHandler()
         parser.add_login_arguments()
-        parser.add_argument(
-            "--file", help=SUPPRESS)
-        parser.add_argument(
-            "command", nargs="?",
-            choices=("new", "update"),
-            help="operation to be performed")
-        parser.add_argument(
-            "Class", nargs="?",
-            help="OMERO model object name, e.g. Project")
-        parser.add_argument(
-            "fields", nargs="*",
-            help="fields to be set, e.g. name=foo")
-        parser.set_defaults(func=self.process)
-
-    def process(self, args):
+        sub = parser.sub()
+
+        new = parser.add(sub, self.new, "Create a new OMERO object")
+
+        update = parser.add(
+            sub, self.update, "Update an existing OMERO object")
+
+        load = parser.add(
+            sub, self.load,
+            "Load a series of commands from a file or the stdin")
+        load.add_argument("file", nargs="*", type=FileType("r"),
+                          default=[sys.stdin])
+
+        graphreport = parser.add(
+            sub, self.graphreport, "Generate")
+        graphreport.add_argument(
+            "--source", help="Configuration file to be used. Default:"
+            " etc/grid/config.xml")
+        graphreport.add_argument(
```

Comment:
Indeed. All sorts of logging guff gets dumped out to the terminal window. That would have to be fixed first.